app/frontend/src/features/auth/AuthPage.tsx

```diff
@@ -135,33 +128,34 @@ export default function AuthPage() {
 
   return (
     <div className={classes.authPage}>
+      {/***** MOBILE ******/}
       <Hidden mdUp>
         <div className={classes.content}>
-          <Typography variant="h1">{INTRODUCTION}</Typography>
-          <Typography classes={{ root: classes.subtitle }}>
-            {COLLABORATIVE}
+          <Typography variant="h1">{INTRODUCTION_TITLE}</Typography>
+          <Typography className={classes.subtitle}>
+            {INTRODUCTION_SUBTITLE}
           </Typography>
           <Divider classes={{ root: authClasses.divider }} flexItem />
           <div className={classes.mobileNavigation}>
             <Button
-              component={Link}
-              to={loginRoute}
               classes={{
                 label: authClasses.buttonText,
-                root: `${authClasses.button} ${classes.button}`,
               }}
+              className={`${authClasses.button} ${classes.button}`}
```

Comment:
I usually use `classnames(authClasses.button, classes.button)` but I don't think it matters if there's nothing conditional.